charts/kubernikus-system/charts/prometheus/templates/_prometheus.yaml.tpl

```diff
@@ -6,7 +6,10 @@ global:
   scrape_timeout: 55s
 
   external_labels:
-    region: k-{{ .Values.global.region }}
+    region: {{ required ".Values.global.region missing" .Values.global.region }}
+    cluster: {{ required ".Values.global.cluster missing" .Values.global.cluster }}
```

Comment:
This is not defined in the current values, or is it somewhere I haven't checked? Also, at least until now it doesn't seem to be used. 

Maybe better to remove it for the sake of this PR?

Reply:
There is a PR waiting in cc/secrets that contains the necessary values.